### PR TITLE
Align Zeitaku logo to the center

### LINKED ISSUE #17: Align Zeitaku logo to the center
The Zeitaku logo is slightly to the right of the container. See if you can align it to the center. 
### PS: This Property
```css
.logo {
    text-align: center;
}
```
### Wont work!! 🙃

### PR DESCRIPTION
**Pull request by Abhijeet-Anand-01**
Hey I successfully managed to center the alignment of Zeitaku logo. Hope it helps! Thankyou!

### CODE REVIEW

**Review by Evavic44 — CHANGES_REQUESTED** [review]
It aligned alright. Nice! but the responsiveness isn't looking good. 
### Recommended Solution
Try just the Transform: translate property so, the logo isn't fixed to the center.
```css
.logo { transform: translateX(-50%);}
```
Then we can align it on other devices using media queries. 
main

File: css/style.css [hunk 1]
```diff
@@ -67,6 +67,10 @@ img {
    font-size: 1.5rem;
    padding: 0.938rem;
    order: 2;
+   position: absolute;
```
Comment (Evavic44) [inline css/style.css]:
```css
.logo { transform: translateX(-50%);}
```

Comment (Abhijeet-Anand-01) [reply css/style.css]: Ohh sorry that's my fault. I'll fix it asap.

P.S. Thanks for the help (It's my first time to Open source contribution) :)

**Comment by Abhijeet-Anand-01** [comment]
Hello! Finally it's done. The logo is aligned to the center perfectly and also I've taken care of the responsiveness this time.

Thankyou!

**Review by Evavic44 — APPROVED** [review]
This is perfect! LGTM! 👍🏾

**Pull request merged**